include/quic/utils.hpp

```diff
@@ -108,6 +108,8 @@ namespace oxen::quic
     inline constexpr size_t MAX_LAZY_PMTUD_UDP_PAYLOAD = MAX_PMTUD_UDP_PAYLOAD;                // 1452
     inline constexpr size_t MAX_GREEDY_PMTUD_UDP_PAYLOAD = (MAX_LAZY_PMTUD_UDP_PAYLOAD << 1);  // 2904
 
+    inline constexpr size_t FLAG_WRITE_MORE = 0x01u;
```

Comment:
This should be set to ngtcp2's constant.  I think for now we have to assume the datagram and stream constants will always equal one another, but they *could* change, and nevertheless it's better to know where the number came from than to have a magic number.